lms/static/sass/shared/_footer.scss

```diff
@@ -301,13 +306,23 @@ $edx-footer-bg-color: rgb(252,252,252);
     @extend %edx-footer-reset;
     @extend %edx-footer-section;
     width: flex-grid(6,12);
+
+    @media print {
+      width: 100%;
+      border: none;
+      padding-right: none;
```

Comment:
Change this rule and the margin rule afterward to use 0 instead of none. This issue resulted in some strange rendering for me where the courseware was much narrower than intended. 

Also, for RTL support use the following:
@include margin-right(0); @include padding-right(0);